### PR TITLE
3.5.2-rc.1

### PR DESCRIPTION
**Pull request by bastianallgeier**

### CODE REVIEW

File: cacert.pem [hunk 1]
```diff
@@ -1,3435 +1,9 @@
...
------END CERTIFICATE-----
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
+<html><head>
+<title>301 Moved Permanently</title>
+</head><body>
+<h1>Moved Permanently</h1>
+<p>The document has moved <a href="https://curl.se/ca/cacert.pem">here</a>.</p>
+<hr>
+<address>Apache Server at curl.haxx.se Port 80</address>
+</body></html>
```
Comment (afbora) [inline cacert.pem]: 🤔 @bastianallgeier

Comment (bastianallgeier) [reply cacert.pem]: Shit. That's not good. @lukasbestle any idea how to fix that? Should we simply revert to the last state?

Comment (afbora) [reply cacert.pem]: Just update URL in `composer.json` line `post-update-cmd` with `https://curl.se/ca/cacert.pem`

Comment (afbora) [reply cacert.pem]: Old URL: https://curl.haxx.se/ca/cacert.pem
New URL: https://curl.se/ca/cacert.pem

Comment (bastianallgeier) [reply cacert.pem]: Ah, I found out how to fix it.

**Review by afbora — APPROVED** [review]

**Pull request merged**